Dart/resources/META-INF/plugin.xml

```diff
@@ -143,6 +143,7 @@
 
     <resolveScopeProvider implementation="com.jetbrains.lang.dart.resolve.DartResolveScopeProvider"/>
 
+    <!--<externalAnnotator language="Dart" implementationClass="com.jetbrains.lang.dart.analyzer.DartAnalysisServerColorAnnotator"/>-->
```

Comment:
Currently DAS only reports error notification for Dart in HTML.
So, no, it does not work.

OTOH, Dart in HTML is used pretty rarely.